packages/dashboard/src/components/Card/ActionBar/ActionBar.types.ts

```diff
@@ -19,6 +19,30 @@ export interface IActionBarStyles {
   root: IStyle;
 }
 
+export interface IAccessability {
```

Comment:
Spelling nit: `IAccessability` => `IAccessibility`

But you might be able to avoid this entirely by just extending native HTML element props which already contain native fields such as `ariaLabel`, `title`, etc.

Reply:
There are some other misspellings of "accessibility" you may want to correct prior to merge.

Reply:
If we extend native HTML element props user might confuse why because am adding accessibility to the action bar overflow button.passing props `actionBarAccessibility`  like this.This prop contain all accessibility props for actionBar Overflow button.

Reply:
> 
> 
> If we extend native HTML element props user might confuse why because am adding accessibility to the action bar overflow button.passing props `actionBarAccessibility` like this.This prop contain all accessibility props for actionBar Overflow button.

I leave the inheritance up to you. However, the spelling comment still stands as it is currently incorrect and important to be done properly since it is an interface (I think).

Reply:
can you please review from this commit `4fdf5b38a5fb563c1d26c921c415cb0d97da3565`.

Reply:
I remove the `IAccessibility ` interface and modify spelling mistakes.